src/app/editor/component/module-variables/module-variables.component.scss

```diff
@@ -1,22 +1,27 @@
 // Copyright 2021 Carnegie Mellon University. All Rights Reserved.
 // Released under a MIT (SEI)-style license. See LICENSE.md in the project root for license information.
 
-.code-editor {
-  border: 2px solid lightgray;
-  height: calc(100vh - 100px);
-  min-height: calc(100vh - 300px);
-  overflow-x: hidden;
-  overflow-y: hidden;
-  .editor-container {
-    height: calc(100vh - 100px);
-    min-height: calc(100vh - 100px);
-  }
-}
-
 h1 {
   display: inline-block;
 }
 
 mat-form-field {
   display: block;
 }
+
+.form-actions {
+  margin-top: 1em;
+}
+
+.changed-hint {
+  color: red !important;
```

Comment:
perhaps use the theme 'warn' color here and in line 22?

Reply:
Good idea, thanks. Not sure if there's a better way than importing the theme in the component scss, but that seems to work.